clients/pubsub.go

```diff
@@ -33,20 +33,20 @@ type PubSub struct {
 }
 
 // NewClient initializes a Pub/Sub client and starts receiving a messages to struct channels.
-func NewClient(ctx context.Context, cfg config.Source) (PubSub, error) {
+func NewClient(ctx context.Context, cfg config.Source) (*PubSub, error) {
 	const maxOutstandingMessages = 1000
```

Comment:
maybe lets move it from function

Reply:
Did you mean to move it to the `Open` method?

Reply:
sorry I mean

const maxOutstandingMessages = 1000

Reply:
ok
lets save it here